PythonScripts/inviteToChannel.py

```diff
@@ -0,0 +1,32 @@
+from flask import jsonify
+import app
+
+
+def addUserToChannel(channel_name, username):
+    # Retrieve user from the database
+    user = app.db.retrieve_from_user(username)
+    channel_information = app.db.retrieve_from_channel(channel_name)
+    
+    #adds user to channel info
+    if channel_information:
+        users = channel_information.get("users",[])
+        if username not in users:  
+            users.append(username)
+            channel_information["users"] = users
+            app.db.update_channel(channel_name,channel_information)
+    
+    if user:
+        # Get list of channels for user, or create an empty list if it doesn't exist
+        channels = user.get("channels", [])
+        if channel_name not in channels:
+            # Add new channel to list
+            channels.append(channel_name)
+            # Update user with new list of channels
+            user["channels"] = channels
+            app.db.update_user(user)  # Update user with new list
```

Comment:
Code seems good with good usage of commenting to make it clear. Seems to be fully functional i see no issues.